patches/chrome-test-base-testing_profile.cc.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/chrome/test/base/testing_profile.cc b/chrome/test/base/testing_profile.cc
+index 83974b28dd687012834a593464fe48a05d03c8bd..19742175491a0322876122d0a2588f1b3dfae788 100644
+--- a/chrome/test/base/testing_profile.cc
++++ b/chrome/test/base/testing_profile.cc
+@@ -143,6 +143,10 @@
+ #include "components/offline_pages/core/stub_offline_page_model.h"
+ #endif
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
header files can be added without patching and `brave::RegisterProfilePrefs` can also be injected into `RegisterUserProfilePrefs` for this file only with a chromium_src override

Reply:
just make sure to include `chrome/browser/prefs/browser_prefs.h` before overriding `RegisterUserProfilePrefs` in the testing_profile.cc chromium_src override

Reply:
didn't get this... I can't see `testing_profile.cc` in `chromium_src`. Could you please help me with this?

Reply:
see the new comment above about overriding browser_prefs.cc instead. The file should be added to chromium_src and then do an override like the others in that directory